packages/extension-chakra-storefront/src/pages/checkout/partials/payment-form.jsx

```diff
@@ -38,54 +39,76 @@ const PaymentForm = ({form, onSubmit}) => {
                                 borderBottom="1px solid"
                                 borderColor="gray.100"
                             >
-                                <Radio value="cc">
-                                    <Flex justify="space-between">
-                                        <Stack direction="row" align="center">
+                                <RadioGroup.Item
+                                    value="cc"
+                                    w="full"
+                                    display="flex"
+                                    alignItems="center"
+                                >
+                                    <RadioGroup.ItemHiddenInput />
+                                    <RadioGroup.ItemIndicator />
+                                    <RadioGroup.ItemText flex="1">
+                                        <Flex justify="space-between">
+                                            <Stack direction="row" align="center">
+                                                <Text fontWeight="bold">
+                                                    <FormattedMessage
+                                                        defaultMessage="Credit Card"
+                                                        id="payment_selection.heading.credit_card"
+                                                    />
+                                                </Text>
+                                                <Tooltip
+                                                    content={formatMessage({
+                                                        defaultMessage:
+                                                            'This is a secure SSL encrypted payment.',
+                                                        id: 'payment_selection.tooltip.secure_payment'
+                                                    })}
+                                                    contentProps={{
+                                                        css: {'--tooltip-bg': 'colors.blue.800'}
```

Comment:
Nit: what do you think we make a constant for tthis at the top of checkout? or do you think it is not worth it?